hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/MasterContext.java

```diff
@@ -582,6 +587,15 @@ private void invokeOnParticipants(Map<MemberInfo, InternalCompletableFuture<Obje
         }
     }
 
+    private ClassLoader swapJobClassLoader(NodeEngine nodeEngine, long jobId) {
```

Comment:
I think it would be better if this took ClassLoader as parameter instead, and make it a static method with name `swapContextClassLoader`